### PR TITLE
HWP-272 Create test file for GET /api/community/{title}

### PR DESCRIPTION
**Pull request by mattsiel**
# Description

This PR includes the following proposed change(s):

-Ticket number 272
- creates a file with the test suite for the get community by title API request

## Requires

Add 'Requires Attention' label if appropriate.

- [ ] Requires an update to the .env file
- [ ] Requires adding a file listed in the .gitignore
- [ ] Requires an opinion or answer to a question, see comments.
- [ ] Requires other attention, see below.

If there are any requirements for the developer to make after this PR is merged please list them here.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

For pull requests that are not just a simple fix, please check that the branch works on your local machine.

- [ ] Tested by Zach.
- [ ] Tested by Brandon.
- [ ] Tested by Brady.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [ ] If attention is required by the developer such as updating the .env file, these requirements have been listed.

### CODE REVIEW

File: app/server/tests/api-getCommunitiesByTitle.test.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('Testing the get community by title function with the "Welcome" communi
 });
 
 
-describe('Testing login for test user', () => {
+describe('Logging in the test user', () => {
```
Comment (akroon3r) [inline app/server/tests/api-getCommunitiesByTitle.test.js:28]:
```suggestion
describe('Logging in as the test user', () => {
```

File: app/server/tests/api-getCommunitiesByTitle.test.js [hunk 2]
```diff
@@ -84,3 +84,19 @@ describe('Testing the get community function with the "Welcome" community after
     expect(response.text).toBe("Forbidden.");
   });
 });
+
+describe('Testing the get community function with the "Welcome" community after logging in, but with null as the token', () => {
```
Comment (akroon3r) [inline app/server/tests/api-getCommunitiesByTitle.test.js:88]: lets keep the description short for this to 'Get Community - Post Login' and maybe keeping the details out in a comment header before line 88.

Thoughts?

Comment (mattsiel) [reply app/server/tests/api-getCommunitiesByTitle.test.js:88]: sure will keep the names shorter 👍

I will go back to the previous test files and rename the tests in my next pull request

Comment (akroon3r) [reply app/server/tests/api-getCommunitiesByTitle.test.js:88]: let me know when you've made the changes to this PR and I'll re-review.

**Pull request closed without merging**